lib/mayaUsd/render/mayaToHydra/renderOverrideUtils.h

```diff
@@ -66,13 +66,17 @@ class HdMayaManipulatorRender : public MHWRender::MSceneRender {
         : MHWRender::MSceneRender(name) {}
 
     MUint64 getObjectTypeExclusions() override {
-        return ~MHWRender::MFrameContext::kExcludeManipulators;
+        return MHWRender::MSceneRender::getObjectTypeExclusions() | ~(kExcludeManipulators | kExcludeHoldOuts);
     }
 
     MHWRender::MClearOperation& clearOperation() override {
         mClearOperation.setMask(MHWRender::MClearOperation::kClearNone);
         return mClearOperation;
     }
+
+    MSceneFilterOption renderFilterOverride() override {
+        return MSceneFilterOption(MHWRender::MSceneRender::renderFilterOverride() | kRenderPostSceneUIItems);
```

Comment:
Can you return kRenderPostSceneUIItems directly? From API doc, MSceneRender::renderFilterOverride() returns kNoSceneFilterOverride which equals 0.